app/backend/src/tests/test_events.py

```diff
@@ -1989,12 +1989,43 @@ def test_RemoveEventOrganizer(db):
         with pytest.raises(grpc.RpcError) as e:
             api.RemoveEventOrganizer(events_pb2.RemoveEventOrganizerReq(event_id=event_id))
         assert e.value.code() == grpc.StatusCode.FAILED_PRECONDITION
-        assert e.value.details() == errors.EVENT_NOT_AN_ORGANIZER
+        assert e.value.details() == errors.EVENT_EDIT_PERMISSION_DENIED
 
         res = api.GetEvent(events_pb2.GetEventReq(event_id=event_id))
         assert not res.organizer
```

Comment:
These were tests that were here before me that started failing once I did the `_can_edit_event` change so please check me if this is right as I don't feel entirely certain.

I think the first one is checking if the person is not an organizer. The second is testing is someone is trying to remove a person who is not an organizer.

Since '_can_edit_event` handles this now, both these fall into that error so it has to be more vague.

At least that's what I'm understanding.